nri/config.go

```diff
@@ -0,0 +1,38 @@
+package nri
```

Comment:
nit: if we want this public, can it go in `pkg`. I'd actually lean towards internal--I wouldnt' expect anyone to import it, so we could then put it in `internal`

Reply:
I moved this to internal/nri.